### ISSUE TITLE
Homepage should indicate instructions on how to join an existing meeting

### ISSUE THREAD

**Issue by pmontecchiarini** [jitsi/jitsi-meet#6913]
The homepage does not provide instructions on how to join an existing conversation.

I think previous to the input to start a new meeting there should be a text indicating that in order to join an existing meeting one must type the URL provided on the navigation bar or create a new meeting by typing the meeting name and pressing GO.

Alternative solutions could be implementing two buttons to either join or create meetings.

**Comment by saghul** [comment]
Thanks for the feedback!

In jitsi those 2 operations are just one. If the meeting didn’t exist before it’s created, otherwise it’s joined.

We should try to find a way to express this better on the welcome page...

**Comment by pmontecchiarini** [comment]
Thankyou for the quick response. In that case, as a user, that was not clear to me. 
But also, I typed the meeting name that was given to me on the input and press go. It opened up a meeting, it did not ask me for a password, and some of my friend where there. But 5 minutes later we realized the teacher was on a parallel meeting. I had to type the exact url she gave me in order to access the meeting. Typing the name on the input from the home page wouldn't do it. Maybe its a different issue. I'm not sure.
